prow/cmd/config-bootstrapper/main.go

```diff
@@ -124,8 +131,24 @@ func main() {
 		if cm.Namespace == "" {
 			cm.Namespace = configAgent.Config().ProwJobNamespace
 		}
-		logger := logrus.WithFields(logrus.Fields{"configmap": map[string]string{"name": cm.Name, "namespace": cm.Namespace}})
-		if err := updateconfig.Update(&osFileGetter{root: o.sourcePath}, client.CoreV1().ConfigMaps(cm.Namespace), cm.Name, cm.Namespace, data, nil, logger); err != nil {
+		if cm.Cluster == "" {
+			cm.Cluster = kube.DefaultClusterAlias
+		}
+		logger := logrus.WithFields(logrus.Fields{"configmap": map[string]string{"name": cm.Name, "namespace": cm.Namespace, "cluster": cm.Cluster}})
+		cmCMI := client.CoreV1().ConfigMaps(cm.Namespace)
+		if buildClusterK8SClients == nil && cm.Cluster != kube.DefaultClusterAlias {
```

Comment:
You don't need this check, just grab the value. It will be `!ok` from a nil map

Reply:
Done.
Good to know of this.